mx.truffleruby/native-image.properties

```diff
@@ -10,6 +10,8 @@ LauncherClass = org.truffleruby.launcher.RubyLauncher
 Args = -H:MaxRuntimeCompileMethods=5400 \
        -H:SubstitutionResources=org/truffleruby/aot/substitutions.json \
        -H:+AddAllCharsets \
+       -H:Path=${.}/bin \
+       -H:Name=truffleruby \
```

Comment:
I think using something other than `truffleruby` would make debugging or looking at logs easier since this is the same name as the non-native image.

Reply:
It's needed, that's what other launchers like irb, gem, etc expect.

Reply:
The Bash launcher is never distributed, and not part of the TRUFFLERUBY-ZIP distribution anymore. It only exists in development. For SVM+JVM, GraalVM deals with it.